### PR TITLE
ramips: added support for Kimax U35WF

### PR DESCRIPTION
**Pull request by hanetzer**
Based on https://github.com/lede-project/source/pull/965

Requested by alazywor1 in freenode/#openwrt

Signed-off-by: Marty E. Plummer <hanetzer@startmail.com>

### CODE REVIEW

**Comment by bobafetthotmail** [comment]
Note: I'm not a core developer so I won't be able to merge this commit, but core developers will very likely require the same things I'm asking, before they merge it.

The commit description lacks the hardware specifications for this device and also lacks the instructions to install OpenWrt in this device.  See the submission guidelines https://openwrt.org/submitting-patches#submitting_patches

Also the default packages included in the image contain far more than the bare minimum needed by this device to work. I think **kmod-usb-storage kmod-usb-storage-extras swap-utils wipefs mount-utils kmod-nls-utf8 kmod-nls-base kmod-fs-btrfs kmod-fs-ext4 kmod-fs-exfat kmod-fs-msdos kmod-fs-ntfs kmod-fs-vfat mkdosfs ntfs-3g fdisk cfdisk hdparm e2fsprogs btrfs-progs block-mount blkid usbutils** should NOT go in the default image. Please remove them.

See the instructions here https://openwrt.org/submitting-patches#squashing_commits on how to modify this PR without closing it and open a new one.

File: target/linux/ramips/dts/U35WF.dts
Comment (mkresin) [inline target/linux/ramips/dts/U35WF.dts]: The whole dts file is kind of a mess:

- whitespace issues
- gpio macros not used
- button macros not used
- basically all nodes defined in mt7620.dtsi are redefined instead of referenced

Please have a look at the existing dts files to see how it is done properly

**Review by mkresin — CHANGES_REQUESTED** [review]
Please use ```git commit --amend``` to add changes to your commit. Use ```git push --force``` to overwrite your github repository/branch. The PR will afterwards update automatically. Please don't close this PR and open a new one.

Expect more change requests. I stopped at the moment I had a closer look at the dts file.

**Comment by mkresin** [comment]
Please reopen the PR if you have the time to address the requested changes.

**Pull request closed without merging**
